ngi_pipeline/conductor/flowcell.py

```diff
@@ -159,11 +157,11 @@ def setup_analysis_directory_structure(fc_dir, projects_to_analyze,
         return []
     # Map the directory structure for this flowcell
     try:
-        fc_dir_structure = parse_casava_directory(fc_dir)
+        fc_dir_structure = parse_xten_directory(fc_dir)
```

Comment:
@mariogiov can we rename the functions to better names: "parse_bcl2fastq1_8_directory" and "parse_bcl2fastq2_5_directory"?